src/main/java/com/verily/genomewarp/utils/GenomeWarpUtils.java

```diff
@@ -58,13 +59,13 @@
   public static final Pattern dnaSequence = Pattern.compile("[ACTGactg]+");
   public static final int VARIANT_CONTEXT_SIZE = 5;
 
-  public static void fail(String message) {
```

Comment:
I'm not sure these logging helpers provide any utility. They are being used inconsistently in the codebase, please either homogenize to use them or just get rid of them and log directly.

Reply:
fail is being used somewhat consistently. I'm removing warn, it has only a single use in the code.